app/controllers/games_controller.rb

```diff
@@ -37,14 +39,24 @@ def update
   end
 
   def destroy
-    Game.destroy(params[:id])
-    redirect_to "/games"
+    @game = Game.find_by(user: current_user, id: params[:id])
+    if !@game.nil?
```

Comment:
If you want to avoid this explicit check for nil, you can use the `find_by!` method instead:

```ruby
def destroy
  @game = Game.find_by!(user: current_user, id: params[:id])
  # or
  @game = current_user.games.find(params[:id])
  # ...
```

If the game can't be found, it will raise an `ActiveRecord::RecordNotFound` exception and Rails will return a 404 response.